content-repo/extra-docs/packs/QRadar.md

```diff
@@ -0,0 +1,148 @@
+---
+id: QRadar
+title: QRadar 
+description: This pack contains all the content objects you need to interact with your QRadar SIEM, including integrations, playbooks, mappers, incident types, layouts, and scripts.
+---
+
+Use this pack's content to ingest offenses from your QRadar system and to run playbooks that perform searches and add indicators to reference sets.
+Although we recommend enabling the integration to fetch offenses from QRadar, you can use the integration even without fetching since this pack has content such as playbooks that are not related to offenses.
+
+Prior to start using the pack we recommend for you to review this detailed [article](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-2/cortex-xsoar-tutorials/tutorials/ingest-incidents-from-a-siem.html) which covers how to configure the integration to fetch incident and get familiarized with working with the generic playbook and incident layout.
```

Comment:
@michalgold WDYT

Reply:
need to make sure it is doc-approved as well. 
Also, need to better emphasize what is in the article and what is in the doc. I think the main difference is that the documentation has more of a use case overview as well.

Reply:
@julieschwartz18 Please review the last editions